dashboard/src/components/dashboard/Dashboard.vue

```diff
@@ -60,6 +60,6 @@ export default {
 
 <style scoped lang="scss">
 .Dashboard {
-	height: 100%;
+	margin-top: 2em;
```

Comment:
```suggestion
	height: calc(100% - 2em);
	margin-top: 2em;
```

I like the LogIn component in the middle of the page.